Examples/Python/tests/test_alignmentdecorator.py

```
@@ -0,0 +1,70 @@
+        logLevel=acts.logging.INFO,
+    )
+    
+    sequencer.addContextDecorator(alignDeco) 
+    sequencer.run()
+    
+test_alignmentdecorator_io_mode()
+
+def test_alignmentdecorator_gen_mode() :
+    """This tests the alignment decorator in generative mode"""
+    alignDecoConfig = AlignmentDecorator.Config()
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)